BedrockServer.cpp

```diff
@@ -869,29 +869,44 @@ void BedrockServer::postPoll(fd_map& fdm, uint64_t& nextActivity) {
     if (!_suppressCommandPort && state == SQLiteNode::SLAVING && (masterVersion != _version)) {
         SINFO("Node " << _args["-nodeName"] << " slaving on version " << _version << ", master is version: "
               << masterVersion << ", not opening command port.");
-        suppressCommandPort(true);
+        suppressCommandPort("wrong master version", true);
```

Comment:
This message is a bit confusing -- how about `master version mismatch`?  The master version is "right", just different.